infrastructure/lib/infrastructure-stack.ts

```diff
@@ -0,0 +1,54 @@
+import * as ec2 from "@aws-cdk/aws-ec2";
+import * as ecs_patterns from "@aws-cdk/aws-ecs-patterns";
+import * as cdk from "@aws-cdk/core";
+import * as ecs from "@aws-cdk/aws-ecs";
+
+export class InfrastructureStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // TODO we need to figure out best practices for standing up the DB
+    // either in the same CF stack, or in a different one.
+    // what's more, we're going to need to figure out how to create users
+    // in it, and then how to get it initialized to have all of the tables
+    // created for us.
+    // we may be able to use this dynamoDB creation example as a starter: https://github.com/aws-samples/aws-cdk-examples/blob/18f3429414cc80223a04a04d0249b12a7ae13cbb/typescript/api-cors-lambda-crud-dynamodb/index.ts#L10-L21
+    var dsnUser = process.env.SECRET_DSN_USER || "TODO_setDsnUser";
+    var dsnPw = process.env.SECRET_DSN_PASSWORD || "TODO_setDsnPassword";
+    var dsnHost = process.env.SECRET_DSN_HOST || ""; // TODO figure out how to ref the RDS
+    var dbType = dsnHost.length > 0 ? "mysql" : "memory";
+
+    const serverPort = 80;
```

Comment:
Why 80? FWIW the load balancer exposes port 80 by default to the outside world and forwards it to whatever you put here